Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -5,168 +5,142 @@
-            inputLabelStackView.addArrangedSubview(operandLabel)
-            allInputStackView.addArrangedSubview(inputLabelStackView)
-            
-            formulaString += inputOperatorLabelText + inputNumberLabelText.replacingOccurrences(of: ",", with: "")
-            resetInputNumberLabel()
-            inputOperatorLabel.text? = labelText
-            allInputScrollView.scrollToBottom()
+            resetOperandLabel()
+            formulaScrollView.scrollToBottom()
         }
     }
     
-    @IBAction func tapEqual(_ sender: UIButton) {
+    @IBAction func tapEqualButton(_ sender: UIButton) {
+        guard operationReady else {
+            return
+        }
+        
         do {
-            if operationReady {
-                tapOperator(sender)
-                
-                var formula = ExpressionParser.parse(from: formulaString)
-                let result = try formula.result()
-                
-                inputNumberLabel.text = makeNumberFormat(for: String(result))
-                operationReady = false
-                
-                resetInputOperatorLabel()
-                resetFormulaString()
-            }
+            tapOperatorButton(sender)
+            
+            var formula = ExpressionParser.parse(from: formulaString)
+            let result = try formula.result()
+            
+            operandLabel.text = makeNumberFormat(for: String(result))
+            operationReady = false
+            
+            resetOperatorLabel()
+            resetFormulaString()
         } catch let error as OperationError {
             switch error {
             case .operandNotEnoughError:
-                print("Operand Not Enough Error")
+                print(OperationError.operandNotEnoughError)
             case .operatorNotEnoughError:
-                print("Operator Not Enough Error")
+                print(OperationError.operatorNotEnoughError)
             case .divideByZeroError:
-                inputNumberLabel.text = "NaN"
+                operandLabel.text = "NaN"
                 operationReady = false
-                print("divide by zero")
+                
+                print(OperationError.divideByZeroError)
             }
         } catch {
-            print("unexpected error")
+            print(CalculatorError.unexpectedError)
         }
     }
     
-    @IBAction func tapChangeSign(_ sender: UIButton) {
-        let hyphenMinus = "-"
+    @IBAction func tapChangeSignButton(_ sender: UIButton) {
+        let operandText = unwrap(operandLabel.text)
         
-        if let numberLabelText = inputNumberLabel.text,
-           numberLabelText != "0" {
-            if numberLabelText.hasPrefix(hyphenMinus) {
-                inputNumberLabel.text = String(numberLabelText.dropFirst())
-            } else {
-                inputNumberLabel.text = hyphenMinus + numberLabelText
-            }
+        if operandText.isZero {
+            return
         }
```

Comment:
```suggestion
        guard operandLabel.text?.isZero == false else {
            return
        }
```

이렇게 사용하면 `unwrap`을 호출해 할당하지 않아도 괜찮을 것 같습니다.

Reply:
tapChangeSignButton 메서드에서 사용하는 convertSign연산프로퍼티를 메서드로 바꿔줬기 때문에 옵셔널 바인딩이 꼭 필요해졌습니다. 그래서 기존 unwrap해주는 부분을 유지했습니다.